### DISCUSSION TITLE
Configuring Black Hole on Mac to record internal audio with Quicktime

### DISCUSSION THREAD

**Discussion by shevlino** [ExistentialAudio/BlackHole#766] [General]
I'm on a brand new Mac with OS 14.  I'm trying to use Quicktime to record internal audio.  In the past I was using Soundflower when recording, but it's not available for a Mac Silicon.  I've downloaded Black Hole and have selected it as the input for my audio, but I can't seem to record.

**Comment by devinroth** [comment]
Did you set BlackHole as the output for your computer? You may want to use a multi-output.

Devin

**Reply by shevlino** [reply]
Ahh...that did the trick.  I had Black Hole set for input, but not output.  Thank you.

**Reply by dianascala** [reply]
Hello, I am new on this forum, sorry if I introduce myself in your discussion, but I noticed the theme of your discussion and it really interests me. Do you know if blackhole permits to record audio with Quicktime from two different application at the same time? Thank you!

**Comment by devinroth** [comment]
Yeah not an issue.

Devin

**Comment by terrysim9339** [comment]
Where on a Mac os is the place to do a "Multi-Ouput setting???" I am trying to record a Zoom call, getting both built in sound and microphone and I need to be able to hear the presenter in the call. My tests so far, I can record good sound, but not hear it simultaneously.
<img width="471" alt="Screenshot 2024-02-15 at 9 56 32 PM" src="https://github.com/ExistentialAudio/BlackHole/assets/160208574/480d928b-e8e9-4d6f-9046-60da02dfa919">
<img width="481" alt="Screenshot 2024-02-15 at 9 56 13 PM" src="https://github.com/ExistentialAudio/BlackHole/assets/160208574/06952c3f-233b-48db-b87e-cf67f241df46">

**Comment by devinroth** [comment]
Check the WIKI.

Devin
